extensions/shellscript/syntaxes/Shell-Unix-Bash.tmLanguage

```diff
@@ -8,11 +8,16 @@
 		<string>bash</string>
 		<string>zsh</string>
 		<string>bashrc</string>
+        <string>zshrc</string>
```

Comment:
I think you can safely remove these since they're added to the package.json which is the source of truth.